cardinal/testutils/testutils.go

```diff
@@ -85,3 +89,18 @@ func AddTransactionToWorldByAnyTransaction(
 
 	_, _ = world.AddTransaction(txID, value, tx)
 }
+
+func GetMessage[In any, Out any](wCtx engine.Context) (*message.MessageType[In, Out], error) {
+	var msg message.MessageType[In, Out]
+	msgType := reflect.TypeOf(msg)
+	tempRes, ok := wCtx.GetMessageByType(msgType)
+	if !ok {
+		return nil, eris.Errorf("Could not find %s, Message may not be registered.", msg.Name())
```

Comment:
ser, i thought you are a %q maxi

Reply:
this was just a copy pasta, i didnt write this

Reply:
can ninja it though